lib/Target/AArch64/AArch64Relocator.cpp

```diff
@@ -25,6 +25,16 @@ using namespace eld;
 
 namespace {
 
+static Relocator::Result emitSignedOrUnsignedRangeOverflow(Relocation &Rel,
+                                                           AArch64Relocator &R,
+                                                           int64_t Value,
+                                                           unsigned Bits) {
+  int64_t Min = llvm::minIntN(Bits);
+  int64_t Max = llvm::maxUIntN(Bits);
```

Comment:
Is it okay to always assume unsigned here?

If `N` is 32, then `llvm::maxUIntN(32)` will return the value `2^32 -1`. On the other hand, `llvm::maxIntN(32)` would be `2^31 -1`. In this case, isn't the `Max` value incorrect for signed relocations?